### DISCUSSION TITLE
Possible to use upload_to_testflight on Ubuntu?

### DISCUSSION THREAD

**Discussion by justin-pierce** [fastlane/fastlane#20069] [Q&A]
Reasoning for wanting to do this: 

- mac runners are 10X as expensive on github actions
- if it's a separate step, I might try to put in checks to make sure builds are uploaded sequentially to avoid concurrent builds being rejected for finishing out of order

Looks like [Transporter](https://help.apple.com/itc/transporteruserguide/en.lproj/static.html) works on "Red Hat Enterprise Linux (64-bit system)" -- not sure if this is a strict requirement or they just want to officially support one specific distro. Couldn't find anything on the fastlane pages saying which platforms it can and can't run on, so figure I'd ask here. 

Thanks

**Comment by Kukurijek** [comment]
As far as I know you can install the ITSM package on any system, even in a container.

**Comment by olivierschmid** [comment]
@justin-pierce Did you try it? I have Fastlane in Docker-Container running within Gilab-CI. But upload to TestFlight does not work yet.

**Reply by justin-pierce** [reply]
Haven't tried it yet -- no longer working on that project
